ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/autocompleteActivityPicklistQuestion.component.ts

```diff
@@ -47,7 +47,9 @@ import { PicklistSortingPolicy } from '../../../services/picklistSortingPolicy.s
 })
 export class AutocompleteActivityPicklistQuestion extends BaseActivityPicklistQuestion implements OnInit, OnDestroy, OnChanges {
     private readonly ngUnsubscribe = new Subject();
+    private readonly DO_NOT_SORT_LIST_STABLE_IDS = ['PRIMARY_CANCER_SELF', 'PRIMARY_CANCER_CHILD'];
 
+    @Input() studyGuid: string;
```

Comment:
we can access config from any sdk component and take studyGuid from it, no need to pass it via input

Reply:
I see.